src/BinSkim.Driver/DriverUtilities.cs

```diff
@@ -45,5 +49,35 @@ private static ConventionBuilder GetConventions<T>()
 
             return conventions;
         }
+
+        public static IList<string> GetInaccessibleSymbolPaths(string symbolPathString)
+        {
+            IList<string> inaccessibleList = new List<string>();
+            if (string.IsNullOrWhiteSpace(symbolPathString)) { return inaccessibleList; }
+
+            string[] symbolServers = Array.FindAll(symbolPathString.Split('*'), path =>
+            path.StartsWith("http://", StringComparison.OrdinalIgnoreCase) ||
+            path.StartsWith("https://", StringComparison.OrdinalIgnoreCase));
+
+            foreach (string symbolServer in symbolServers)
+            {
+                try
+                {
+                    var httpClient = new HttpClientWrapper();
```

Comment:
@shaopeng-gh Wanted to cross-check that this invocation does not return a new `HttpClient` in each iteration. If so, we should create a single client and re-use that.